crates/proto/src/rustls/tls_stream.rs

```diff
@@ -70,10 +70,12 @@ pub fn tls_from_stream<S: DnsTcpStream>(
 /// # Arguments
 ///
 /// * `name_server` - IP and Port for the remote DNS resolver
+/// * `bind_addr` - IP and port to connect from
 /// * `dns_name` - The DNS name,  Subject Public Key Info (SPKI) name, as associated to a certificate
 #[allow(clippy::type_complexity)]
 pub fn tls_connect<S: Connect>(
     name_server: SocketAddr,
+    bind_addr: Option<SocketAddr>,
```

Comment:
I'm trying to decide if we need a new method hear to avoid the breaking change.

Reply:
I've added tls_connect_with_bind_addr.